core/domain/topic_domain.py

```diff
@@ -491,6 +491,10 @@ def require_valid_name(cls, name):
         if name == '':
             raise utils.ValidationError('Name field should not be empty')
 
+        if len(name) > 35:
+            raise utils.ValidationError(
+                'Topic name should be atmost 35 characters.')
```

Comment:
Here and elsewhere: "at most", not "atmost".

Reply:
Done